### DISCUSSION TITLE
How to use a virtual color with recipes?

### DISCUSSION THREAD

**Discussion by asterikx** [chakra-ui/panda#890] [Q&A] [answered]
How can I use a [virtual color](https://panda-css.com/docs/guides/virtual-color) with recipes (to apply the virtual color to two or more "parts" of a composed component), and mimic Chakra UIs `colorScheme` prop for multipart components?

Relates to #882.

**Comment by astahmer** [comment, marked as answer]
here you go, with support for light/dark color mode as well 😄 
![Screenshot 2023-07-01 at 00 21 49](https://github.com/chakra-ui/panda/assets/47224540/5bb725d3-5cde-42a9-9ae4-2c3717d2230a)


https://play.panda-css.com/dyQcs3w519
